vendor/vendor.json

```diff
@@ -1496,6 +1496,13 @@
 			"revision": "363e3494bdafc50c3447e7a73d94e4caf9fc3c8c",
 			"revisionTime": "2019-10-18T23:19:20Z"
 		},
+		{
+			"checksumSHA1": "wgane+Jx/akhNYEAqL/TlxwvkYg=",
+			"path": "github.com/elastic/go-elasticsearch/v8",
```

Comment:
The `go-elasticsearch` client is intended to be used with go modules. Since APM Server doesn't support that yet, I have to workaround this by running:
```
git clone --branch master https://github.com/elastic/go-elasticsearch.git $GOPATH/src/github.com/elastic/go-elasticsearch/v8
govendor add github.com/elastic/go-elasticsearch/v8/^
```

The license file is now located under the `v8/` folder, which creates an issue with our license checker. I manually copied the license one folder up to prevent this. 
I now updated to actually move the license to the root folder of the client, which ensures it only shows up once in the NOTICE file. 

This is not a nice workaround, but the best I could come up with. Happy to try out other suggestions if there are any.

Reply:
thanks, i deff don't have any better ideas. if we have to do the same process again, maybe a make target for it would help, or document it somewhere (even in the Readme, just add a section "how to update the Go Elasticsearch client")